### PR TITLE
Revise technical overview

### PR DESCRIPTION
**Pull request by banker**

### CODE REVIEW

**Comment by bmansheim** [comment]
Looks good. I try to use one per sentence to make it easier to read in the source. In general I use title case for ## headings and sentence case for ### and below.

Do we want to use the term UI or something like Admin Console? Maybe we talked about this before but it hit me again because I feel like UI minimizes the graphical interface, whereas we want customers to know that the user-friendliest way to interact with our product is with the UI whereas the CLI and API are minimalist interfaces for programmatic interactions. No?

File: content/rs/technology-behind-redis-enterprise.md [hunk 1]
```diff
@@ -1,102 +1,73 @@
...
-to the application itself.
-
-The proxies also support Memcached's text and binary protocols and
-perform in-flight translation between Memcached and Redis protocol
-semantics. This allows Memcached users to enjoy many of the features
-that are not available with open source Memcached, such as built-in
-replication, auto-failover, data persistence backups and scaling
-(out/in) without losing data.
+
+Redis Enterprise was built to simplify operations while providing seamless scalability, top performance, and high availability.
```
Comment (bmansheim) [inline content/rs/technology-behind-redis-enterprise.md:9]: "was built" sounds old to me. Maybe "uses advanced technology"?

File: content/rs/technology-behind-redis-enterprise.md [hunk 2]
```diff
@@ -1,102 +1,73 @@
...
+
+Here are some of the features and design decisions that make this possible.
+
+## Data access and cluster management
+
+A Redis Enterprise Cluster separates data access from cluster
```
Comment (bmansheim) [inline content/rs/technology-behind-redis-enterprise.md:15]: s/Cluster/cluster/

File: content/rs/technology-behind-redis-enterprise.md [hunk 3]
```diff
@@ -1,102 +1,73 @@
...
+
+You can use any standard Redis client to access a Redis Enterprise database via the proxy. The proxies also support cluster-aware clients for applications requiring the lowest possible latency.
+
+In addition, the proxies understand Memcached's text and binary protocols. Applications using Memcached can immediately start using Redis Enterprise and take advantage of its many features, including built-in replication, auto-failover, data persistence, backups, and linear scaling.
 
 ### Cluster management
 
-The cluster manager is a sophisticated governing function that provides
-capabilities such as resharding, rebalancing, auto-failover,
+This cluster manager consists of a number of software components that monitor
```
Comment (bmansheim) [inline content/rs/technology-behind-redis-enterprise.md:29]: "This cluster manager" where was it defined before? Looks to me like it should be "The cluster manager"

File: content/rs/technology-behind-redis-enterprise.md [hunk 4]
```diff
@@ -1,102 +1,73 @@
...
-    performance, including just-in-time pipelining, socket connections,
-    connection pooling, and multiplexing.
-- Improvements to AOF data persistence and optimized rewrite
+persistence configuration, backup, and recovery.
+
+## High performance and availability
+
+Redis Enterprise uses several techniques to optimize performance and availability.
+
+- The cluster is a shared-nothing architecture, which increases reliability and availability, and makes it easy to add and remove nodes.
```
Comment (bmansheim) [inline content/rs/technology-behind-redis-enterprise.md:40]: Maybe: The cluster uses a shared-nothing architecture that increases reliability and availability. This architecture also makes it easy to add and remove nodes.

File: content/rs/technology-behind-redis-enterprise.md [hunk 5]
```diff
@@ -1,102 +1,73 @@
...
-interface, CLI tools, and APIs. Special experience or knowledge is not
-required to create, manage and monitor Redis clusters, nodes, and
-databases in RS.
-
-You can read more about the architecture of RS in [Concepts and
-Architecture]({{< relref "/rs/concepts/_index.md" >}}).
+The symmetric, shared-nothing cluster architecture allows for a variety
+of underlying infrastructures. Each cluster node can be physical or virtual, with variations in memory, storage devices, and number of compute cores. This freedom of configuration allows you to roll out clusters that most efficiently use your resources.
+
+For example, a cluster may be based on a few permanent bare-metal servers with
```
Comment (bmansheim) [inline content/rs/technology-behind-redis-enterprise.md:61]: s/may/can/

File: content/rs/technology-behind-redis-enterprise.md [hunk 6]
```diff
@@ -1,102 +1,73 @@
...
+additional virtual servers for scaling out when load increases
+or decreases.
+
+### Monitoring and alerting
+
+Monitoring and alerting are built-in and configurable. You can use the included dashboards to monitor your cluster, and you can easily [integrate with Nagios, Prometheus, and Grafana]({{< relref "/rs/administering/monitoring-metrics/_index.md" >}}).
```
Comment (bmansheim) [inline content/rs/technology-behind-redis-enterprise.md:67]: s/, and you can/. You can also/

File: content/rs/technology-behind-redis-enterprise.md [hunk 7]
```diff
@@ -1,102 +1,73 @@
...
+
+
+### UI, APIs, and CLIs
+
+You can administer a Redis Enterprise Software Cluster using the web-based UI,
```
Comment (bmansheim) [inline content/rs/technology-behind-redis-enterprise.md:72]: I don't think that Cluster is part of the proper noun.

File: content/rs/technology-behind-redis-enterprise.md [hunk 8]
```diff
@@ -1,102 +1,73 @@
...
+the cluster REST API, and various CLI-based tools. Once the cluster is up and running, you don't need any special experience or knowledge to create, manage, and monitor your databases and cluster nodes.
```
Comment (bmansheim) [inline content/rs/technology-behind-redis-enterprise.md:73]: Well it's not as easy as a simple microwave but I understand that the point is that the actions are simple.

File: content/rs/technology-behind-redis-enterprise.md (hunk 1 above)
Comment (banker) [reply content/rs/technology-behind-redis-enterprise.md:9]: "uses advanced technology" sounds corny and vague to me.

File: content/rs/technology-behind-redis-enterprise.md (hunk 4 above)
Comment (banker) [reply content/rs/technology-behind-redis-enterprise.md:40]: I took "uses", which is definitely better.

File: content/rs/technology-behind-redis-enterprise.md (hunk 6 above)
Comment (banker) [reply content/rs/technology-behind-redis-enterprise.md:67]: Not seeing what's wrong the with compound sentence here. Starting two consecutive sentences with "You can" feels choppy to me.

File: content/rs/technology-behind-redis-enterprise.md (hunk 8 above)
Comment (banker) [reply content/rs/technology-behind-redis-enterprise.md:73]: That text was there before. I'm removing it. Agreed.

**Pull request merged**